pkg/apis/v1alpha5/suite_test.go

```diff
@@ -567,6 +570,84 @@ var _ = Describe("Provisioner Annotation", func() {
 		}
 		provisioner = test.Provisioner(testProvisionerOptions)
 	})
+	DescribeTable(
+		"Static Hash Values",
```

Comment:
nit: I think the test gets prefixed with each container that wraps it so if you find yourself repeating `"should match static hash values"`, you may just want to push that up to the container-level or remove it entirely

Reply:
I've changed it in the latest commit to just be `"should match with ..."`. I didn't want to move that up a level since it would then read `static hash values should match with [It] ...` in the `go test` logs, rather than `Static Hash Values [It] should match with ...`.